code/modules/GOAP/monkey/actions.dm

```diff
@@ -0,0 +1,350 @@
+/datum/goap_action/monkey/throwitem
+	name = "Throw item"
+	cost = 1
+	cooldown = 20
+
+/datum/goap_action/monkey/throwitem/New()
+	..()
+	preconditions = list()
+	preconditions["grabbedWeapon"] = TRUE
+	preconditions["hasItem"] = TRUE
+	preconditions["hasHumanItem"] = TRUE
+	preconditions["GUN"] = FALSE // don't throw your gun!!
+	effects = list()
+	effects["replaceItem"] = TRUE
+
+/datum/goap_action/monkey/throwitem/AdvancedPreconditions(mob/living/carbon/monkey/C, list/worldstate)
+	var/obj/item/X = locate() in C.held_items
+	if(!X)
+		return FALSE
+	for(var/mob/living/carbon/H in view(5, C))
+		if(H && !H.stat)
+			if(C.should_target(H))
+				target = H
+				break
+	if(!target)
+		target = get_ranged_target_turf(C, C.dir, 10)
+	return (target != null)
+
+/datum/goap_action/monkey/throwitem/RequiresInRange(atom/agent)
+	return FALSE
+
+/datum/goap_action/monkey/throwitem/Perform(mob/living/carbon/monkey/C)
+	var/obj/item/X = locate() in C.held_items
+	if(X)
+		X.forceMove(get_turf(C))
+		X.throw_at(target, 10, 1, C)
+		C.best_force = 0
+		action_done = TRUE
+	else
+		action_done = TRUE
+		return FALSE
+	return ..()
+
+/datum/goap_action/monkey/throwitem/CheckDone(atom/agent)
+	return action_done
+
+/datum/goap_action/monkey/GetItem
+	name = "Grab an item"
+	cost = 2
+
+/datum/goap_action/monkey/GetItem/New()
+	..()
+	preconditions = list()
+	preconditions["hasItem"] = FALSE
+	effects = list()
+	effects["hasItem"] = TRUE
+
+/datum/goap_action/monkey/GetItem/AdvancedPreconditions(mob/living/carbon/monkey/C, list/worldstate)
+	if(C.IsDeadOrIncap() || C.restrained())
+		return FALSE
+	for(var/obj/item/I in oview(4, C))
+		if(I && !C.blacklistItems[I] && !HAS_TRAIT(I, TRAIT_NODROP))
+			if(I.force > C.best_force)
+				C.best_force = I.force
+				target = I
+	return (target != null)
+
+/datum/goap_action/monkey/GetItem/RequiresInRange(atom/agent)
+	return TRUE
+
+/datum/goap_action/monkey/GetItem/Perform(mob/living/carbon/monkey/C)
+	C.drop_all_held_items() // who cares about these items, i want that one!
+	C.equip_item(target)
+	action_done = TRUE
+	return ..()
+
+/datum/goap_action/monkey/GetItem/CheckDone(atom/agent)
+	return action_done
+
+/datum/goap_action/monkey/pickpocket
+	name = "Pickpocket Item"
+	cost = 1
+
+/datum/goap_action/monkey/pickpocket/New()
+	..()
+	preconditions = list()
+	preconditions["hasHumanItem"] = FALSE
+	preconditions["enemyAttack"] = FALSE
+	effects = list()
+	effects["hasHumanItem"] = TRUE
+
+/datum/goap_action/monkey/pickpocket/AdvancedPreconditions(mob/living/carbon/monkey/C, list/worldstate)
+	if(C.IsDeadOrIncap() || C.restrained())
+		return FALSE
+	for(var/mob/living/carbon/human/H in oview(4, C))
+		if(H)
+			for(var/obj/item/I in H.held_items)
+				if(I && !C.blacklistItems[I] && I.force > C.best_force)
```

Comment:
so monkeys won't try to pickpocket most guns once they have a sharp stick?

Reply:
Pickpocketting isn't really meant for combat, its what they do to annoy you they'll try to disarm you in combat if they don't have a weapon already